dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -90,13 +90,10 @@ class SelectiveUnitTestTypes(Enum):
 
 ALLOWED_TEST_TYPE_CHOICES = [
     "All",
-    "Always",
     *all_selective_test_types(),
     "Helm",
     "Postgres",
     "MySQL",
-    "Integration",
```

Comment:
@potiuk I have removed the duplicate TEST_TYPE here from `global_constants.py` file so that test_type doesn't show duplicate entries. 

https://github.com/apache/airflow/blob/main/BREEZE.rst#running-unit-integration-tests-in-groups
Here in screenshot of command, the `--test-type` option contains duplicate. Am i right in thinking that it should not contain duplicate?

Reply:
yep